src/app/ng2-ui-auth/README.md

```diff
@@ -0,0 +1,4 @@
+Files in this directory are originally from https://github.com/ronzeidman/ng2-ui-auth
```

Comment:
I can't find a license in https://github.com/ronzeidman/ng2-ui-auth, although at least one linked repo is MIT-licensed.   Should probably confirm that we're allowed to use it.

Will return and do a deeper review later...